source/_integrations/modbus.markdown

```diff
@@ -682,6 +682,40 @@ climates:
         value 1 is written."
       required: false
       type: integer
+    swing_mode_register:
+      description: "Configuration of the register for Swing mode"
+      required: false
+      type: map
+      keys:
+        address:
+          description: "Address of Swing mode register. (int to call write_register, list of 1 int to call write_registers)"
```

Comment:
The register is a fixed type?

Reply:
2 lines after that at line 693 you can see ` type: [integer, list]`

Reply:
No which request are used to read the register.

Reply:
> No which request are used to read the register.

Uhm.. Actually the integration uses "holding" also for fan mode, hvac mode and, obviously in this case also for swing mode without a clear declaration of that.  
Do you mean that I should clarify this into the documentation and the same for the others?!